src/theme/css/chrome.css

```diff
@@ -465,7 +464,7 @@ ul#searchresults span.teaser em {
     left: calc(var(--sidebar-resize-indicator-width) * -1);
     right: unset;
 }
-.js .sidebar .sidebar-resize-handle {
+.sidebar .sidebar-resize-handle {
```

Comment:
This change results in the sidebar handle being visible and having a cursor, but it doesn't actually work.

Reply:
Ah indeed. Good catch!